Adding in generic update instructions and fixing some links. (#99)
timescale/docs: timescaledb/how-to-guides/update-timescaledb/update-timescaledb-2.md

<highlight type="warning">
These instructions are only for upgrading TimescaleDB 1.x to TimescaleDB 2.0

>  If you need to upgrade your existing TimescaleDB to a newer version in the 
>  release line (eg. 1.7.2->1.7.4 or 2.1->2.2), please see [Update TimescaleDB](/timescaledb/latest/how-to-guides/update-timescaledb/update-timescaledb/)
>  for general update instructions.

Reviewer: ```suggestion
 To upgrade your existing TimescaleDB within the same major version
 (for example, from 1.7.2 to 1.7.4, from 2.1 to 2.2), see [Update TimescaleDB](/timescaledb/latest/how-to-guides/update-timescaledb/update-timescaledb/)
 for general update instructions.
```

</highlight>

### Notice of breaking changes from TimescaleDB 1.3+
TimescaleDB 2.0 supports **in-place updates** just like previous releases. During 

> the update, scripts will automatically configureupdated features to work as expected 

Reviewer: ```suggestion
the update, scripts will automatically configure updated features to work as expected 
```
